### PR TITLE
svg_loader: postpone cloneNode()

### PR DESCRIPTION
**Pull request by mmaciola**
Some svg export software puts `<defs>` element at the end of the file.
If so, the `<defs>` element won't be found, when parsing `<use>`.
In such scenario, this patch postpone node cloning until the whole file
is parsed.

@issue: #568

### CODE REVIEW

File: src/loaders/svg/tvgSvgLoaderCommon.h
Comment (hermet) [inline src/loaders/svg/tvgSvgLoaderCommon.h:317]: It looks too fragmented memory.
How about  Array<SvgNodeIdPair>?

**Review by hermet — CHANGES_REQUESTED** [review]
Please check a comment.

File: src/loaders/svg/tvgSvgLoaderCommon.h
Comment (mmaciola) [reply src/loaders/svg/tvgSvgLoaderCommon.h:317]: Could you describe more?

**Review by JSUYA — APPROVED** [review]
I don't think there are any issues with this patch. I wait for hermet's approval.

File: src/loaders/svg/tvgSvgLoaderCommon.h
Comment (hermet) [reply src/loaders/svg/tvgSvgLoaderCommon.h:317]: @mmaciola sorry for this, \< , \> these tags are ignored by this github viewer. see again the above.

Comment (mmaciola) [reply src/loaders/svg/tvgSvgLoaderCommon.h]: ok. done

**Review by hermet — APPROVED** [review]

**Comment by hermet** [comment]
@mmaciola thanks.

**Pull request merged**
